OmniBLE/OmnipodCommon/BasalDeliveryTable.swift

```diff
@@ -159,6 +159,24 @@ extension BasalDeliveryTable: CustomDebugStringConvertible {
     }
 }
 
+// Normalize basal rate by rounding down to pulse size boundary,
+// but basal rates within a small delta will be rounded up.
+func normalizedBasalRate(rate: Double) -> Double
+{
+    let delta = 0.01
+    return Pod.supportedBasalRates.last(where: { $0 <= rate + delta }) ?? 0
+}
+
+// Return normalized basal rate for pulse timing purposes.
+// For non-Eros, returns the nearZeroBasalRate (0.01) for a zero basal rate.
+func normalizedBasalTimingRate(rate: Double) -> Double {
+    var nrate = normalizedBasalRate(rate: rate)
+    if nrate == 0.0 {
+        nrate = Pod.zeroBasalRate // will be an adjusted value for non-Eros cases
```

Comment:
Eros should not have 0 in the supportedBasalRates, so normalizedBasalRate should never return a 0, right?

Reply:
"Eros should not have 0 in the supportedBasalRates" -> yes
"so normalizedBasalRate should never return a 0, right?" ->  actually no
Because of the way normalizedBasalRate() was written, it would still return a 0 even if a 0 wasn't in supportedBasalRates[] for Eros (see commenting on OmnipodPumpManager.roundToSupportedBasalRate()).

In any case, this wasn't obvious. Will add explicit comment on the renamed normalizedBasalRate() function stating that it will round down to 0 for both Eros and non-Eros pods and will rewrite this function to not use Pod.supportedBasalRates[] to hopefully make this even more obvious.